examples/publish subscribe/bin/StreetlightController.js

```diff
@@ -0,0 +1,46 @@
+const {AvailableEvents, NatsAsyncApiClient, TestClient, TurnonCommandMessage} = require("nats-client");
```

Comment:
Can you replace tabs with spaces here? Github is showing me tabs as 8 spaces long 😅